bots/run-queue

```diff
@@ -64,8 +63,7 @@ def consume_webhook_queue(channel, amqp):
         action = request['action']
         # scan for opened, body changes (edited), and the bots label
         if action in ['opened', 'labeled'] or (action == 'edited' and 'body' in request.get('changes', {})):
-            issue = request['issue']
-            cmd = ['bots/issue-scan', '--issue-data', json.dumps(issue), '--amqp', amqp]
+            cmd = ['bots/issue-scan', '--issues-data', json.dumps(request), '--amqp', amqp]
```

Comment:
So it now passes on the full payload. This makes sense (and I think it'd be cleaner and more flexible for tests-scan, too). I just wonder how this affects the hack that we did in PR #12231, I think that could be reverted now?

Reply:
I'm not actually sure. Since at that point the issue object should be the same. I tihnk the split there comes from it being either an issue or a pull request object. I'll update the comment to reflect that.